### PR TITLE
notes about source-map-support

### PR DESCRIPTION
**Pull request by fabiosantoscode**
Add some notes about source-map-support module in our README and issue template.

### CODE REVIEW

**Comment by kzc** [comment]
Can you please add this to the issue template?
```patch
--- a/.github/ISSUE_TEMPLATE.md
+++ b/.github/ISSUE_TEMPLATE.md
@@ -11,5 +11,6 @@
 <!--
     Complete valid ECMAScript code exhibiting the issue with
-    `terser` alone - without third party tools or libraries.
+    `terser` ALONE - without third party tools or libraries.
+    This means no webpack, rollup, browserify or parcel.
     Ideally the input should be as small as possible.
     Code must be in text form - not a screen cap.
```

File: README.md [hunk 1]
```diff
@@ -358,6 +358,8 @@ identify mistakes like writing mangled keys to storage.
 
 # API Reference
 
+**note:** In the terser CLI we use [source-map-support](https://npmjs.com/source-map-support) to produce good error stacks. In your own app, you're expected to enable source-map-support (read their docs) to have nice stack traces that will make good issues.
```
Comment (kzc) [inline README.md]: Please uppercase `note:`

**Comment by kzc** [comment]
Could you please put the `source-map-support` doc note at the bottom of the README in new a section labelled "Reporting Issues"? Additional info to debug issue can be included there, and the issues template could link to it.

**Comment by fabiosantoscode** [comment]
Pushed an update, check again @kzc

**Comment by kzc** [comment]
Whatever you decide is fine with me, @fabiosantoscode. Take the project in any direction you want. All the best.

**Comment by fabiosantoscode** [comment]
All the best

**Pull request merged**